earth_enterprise/rpms/buildSrc/src/main/groovy/org/opengee/shell/GeeCommandLine.groovy

```diff
@@ -0,0 +1,66 @@
+package org.opengee.shell
+
+class GeeCommandLine {
+    class ExpectedResult {
+        Integer exitCode = null
+        String stdOut = null
+        String stdErr = null
+
+        ExpectedResult(Integer exitCode, String stdOut, String stdErr) {
+            this.exitCode = exitCode
+            this.stdOut = stdOut
+            this.stdErr = stdErr
+        }
+    }
+
+    // Returns the standard output that results from executing a given command.
+    //     To pipe to standard input set `stdInAction` to a lambda that
+    // accepts the standard input writer as an parameter.
+    //     To suppress known command results (return `null`), use the
+    // `suppressedResults` parameter.
+    static def expand(
```

Comment:
Expand seems like an odd name for this method. Isn't it executing the command?

Reply:
Yeah, it's executing it, and returning the standard output.